lib/commands/connect/regions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const cli = require('heroku-cli-util');
+
+const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney'];
```

Comment:
Since we're planning to be part of the region vetting process then for data (our dependency) I would think we're going to be early entries into any region that gets publicly added.